doc/how_to/state/url.md

```diff
@@ -25,35 +29,53 @@ When starting a server session Panel will attach a `Location` component which ca
 
-By default the current [query parameters](https://en.wikipedia.org/wiki/Query_string) in the URL (specified as a URL suffix such as `?color=blue`) are made available on `pn.state.location.query_params`. To make working with query parameters straightforward the `Location` object also provides a `sync` method which allows syncing query parameters with the parameters on a `Parameterized` object.
+By default the current [query parameters](https://en.wikipedia.org/wiki/Query_string) in the URL (specified as a URL suffix such as `?color=blue`) are made available on `pn.state.location.query_params`.
+
+## Sync and Unsync
+
+To make working with query parameters straightforward the `Location` object provides a `sync` method which allows syncing query parameters with the parameters on a `Parameterized` object.
 
 We will start by defining a `Parameterized` class:
 
 ```python
-import param
 import panel as pn
+import param
 
-class QueryExample(param.Parameterized):
-
-    integer = param.Integer(default=None, bounds=(0, 10))
 
+class Settings(param.Parameterized):
+    integer = param.Integer(default=1, bounds=(0, 10))
     string = param.String(default='A string')
+
+    dont_sync = param.String(default='A string')
 ```
 
-Now we will use the `pn.state.location` object to sync it with the URL query string (note that in a notebook environment `pn.state.location` is not initialized until the first plot has been displayed). The sync method takes the Parameterized object or instance to sync with as the first argument and a list or dictionary of the parameters as the second argument. If a dictionary is provided it should map from the Parameterized object's parameters to the query parameter name in the URL:
+Now we will use the `pn.state.location` object to sync it with the URL query string. The sync method takes the **`Parameterized` class or instance** to sync with as the first argument and a list or dictionary of the parameters as the second argument. If a dictionary is provided it should map from the Parameterized object's parameters to the query parameter name in the URL:
 
 ```python
-pn.state.location.sync(QueryExample, {'integer': 'int', 'string': 'str'})
+settings = Settings()
+
+pn.state.location.sync(settings, {'integer': 'int', 'string': 'str'})
 ```
 
-Now the Parameterized object is bi-directionally linked to the URL query parameter, if we set a query parameter in Python it will update the URL bar and when we specify a URL with a query parameter that will be set on the Parameterized, e.g. let us set the 'integer' parameter and watch the URL in your browser update:
+Now the Parameterized object is bi-directionally linked to the URL query parameter.
 
-```python
-QueryExample.integer = 5
+Lets try to serve it as an app
+
+```Python
+pn.Param(settings).servable()
 ```
 
-Note to unsync the Parameterized object you can simply call `pn.state.location.unsync(QueryExample)`.
+<video controls="" poster="../../_static/location_example_app.png">
```

Comment:
Please start putting these in `_static/images` in future. For existing ones I'll do that all at once in another PR at some point.